CMakeLists.txt

```diff
@@ -557,14 +617,23 @@ target_include_directories(notcurses-tester
   PRIVATE
     include
     "${PROJECT_BINARY_DIR}/include"
+    "${TERMINFO_INCLUDE_DIRS}"
     src/lib
 )
 target_link_libraries(notcurses-tester
   PRIVATE
     unistring
     notcurses++
     "${TERMINFO_LIBRARIES}"
+    doctest::doctest
 )
+if(APPLE)
+  target_link_directories(notcurses-tester
+    PRIVATE
+      "${TERMINFO_LIBRARY_DIRS}"
```

Comment:
likewise, oughtn't this be done for everyone?

Reply:
Yeah, perhaps we should do that.